packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.types.ts

```diff
@@ -118,6 +123,8 @@ export type ITeachingBubbleStyleProps = Required<Pick<ITeachingBubbleProps, 'the
   Pick<ITeachingBubbleProps, 'hasCondensedHeadline' | 'hasSmallHeadline' | 'isWide'> & {
     /** Class name for callout. */
     calloutClassName?: string;
+    /** Class name for multistep text */
+    multistepTextClassName?: string;
```

Comment:
I don't see this used or passed on anywhere. If we need to make `footerContent` styleable it should have a `footerContent` entry in the `ITeachingBubbleStyles` interface. However, I'm hoping that's not needed since we already have `footer`.